src/phoenix/experimental/evals/functions/classify.py

```diff
@@ -116,8 +116,8 @@ def llm_classify(
         run_sync (bool, default=False): If True, forces synchronous request submission. Otherwise
         evaluations will be run asynchronously if possible.
 
-        concurrency (int, default=20): The number of concurrent evals if async submission is
-        possible.
+        concurrency (Optional[int], default=None): The number of concurrent evals if async
+        submission is possible. If not set, the default is determined on a per-model basis.
```

Comment:
```suggestion
        concurrency (Optional[int], default=None): The number of concurrent evals if async
        submission is possible. If not provided, a recommended default concurrency is set on a per-model basis.
```

Suggestion: Make it clear that the default value is the recommended value.

Reply:
This makes me wonder if we really should allow the user to use many different models inside of `run_evals`.